pyproject.toml

```diff
@@ -39,6 +39,51 @@ version_variables = [
 branch = "main"
 build_command = "pip install poetry && poetry build"
 
+[tool.pdm]
+[tool.pdm.dev-dependencies]
```

Comment:
Why is pdm added here? this library is managed using poetry currently. We should stick with a single build tool / dependency manager

Reply:
Yeah I forgot we were using poetry for this repo 😅  easy to get confused with different build systems

Reply:
fixed